src/main/java/org/opentripplanner/transit/service/TransitModel.java

```diff
@@ -55,7 +55,28 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Repository for Transit entities.
+ * The TransitModel groups together all instances making up OTP's primary internal representation
+ * of the public transportation network. Although the names of many entities are derived from
+ * GTFS concepts, these are actually independent of the data source from which they are loaded.
+ * Both GTFS and NeTEx entities are mapped to these same internal OTP entities. If a concept exists
+ * in both GTFS and NeTEx, the GTFS name is used in the internal model. For concepts that exist
+ * only in NeTEx, the NeTEx name is used in the internal model.
+ *
+ * A TransitModel instance also includes references to some transient indexes of its contents, to
+ * the TransitLayer derived from it, and to some other services and utilities that operate upon
+ * its contents.
+ *
+ * The TransitModel stands in opposition to two other aggregates: the Graph (representing the
+ * street network) and the TransitLayer (representing many of the same things in the TransitModel
+ * but rearranged to be more efficient for Raptor routing).
+ *
+ * At this point the TransitModel is not often read directly. Many requests will look at the
+ * TransitLayer rather than the TransitModel it's derived from. Both are often accessed via the
+ * TransitService rather than directly reading the fields of TransitModel or TransitLayer.
+ *
+ * TODO RT_AB: consider renaming. By some definitions this is not really the model, but a top-level
+ *   object grouping together instances of model classes with things that operate on and map those
+ *   instances.
```

Comment:
I have explained what I meant on another comment (seeing a "model" as a non-instantiable idea). But this might be due to me not understanding some standardized OOP usage of terminology, will read up on this myself.